MonstieWash/Assets/Scripts/Tutorial/TutorialCounter.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class TutorialCounter : MonoBehaviour
+{
+    private TextMeshProUGUI m_Text;
+    private TutorialManager m_Manager;
```

Comment:
### 🗒️ **Convention** | _This is for maintaining our coding conventions._

Just no capitals needed for these.

```suggestion
    private TextMeshProUGUI m_text;
    private TutorialManager m_manager;
```